style.css

```diff
@@ -1,11629 +1,17 @@
-@charset "UTF-8";
```

Comment:
There is a conflict in that file due to outdated dependencies , noeasy  fix possible at this time